### PR TITLE
Integrations pages

### PR DESCRIPTION
**Pull request by elad2412**
- 3 new Integration pages

### CODE REVIEW

File: src/lib/components/embla/Embla.svelte [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+<style>
+    .embla {
+        overflow: hidden;
+        position: relative;
+    }
+
+    .embla__container {
+        display: flex;
+    }
+</style>
```
Comment (TorstenDittmann) [inline src/lib/components/embla/Embla.svelte:168]: feel free to use sass here, you also dont need to care about css scopes with svelte and can just use `.container` 👍🏻

File: src/lib/components/embla/EmblaSlide.svelte [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<li class="embla__slide web-carousel-item">
+    <div class="embla__slide__number">
+        <slot />
+    </div>
+</li>
+
+<style lang="scss">
+    @use '$scss/abstract' as *;
+
+    .embla__slide {
```
Comment (TorstenDittmann) [inline src/lib/components/embla/EmblaSlide.svelte:10]: same as here, with svelte you can just do `.slide` or even `div` or `li`

File: src/lib/components/embla/Embla.svelte
Comment (TorstenDittmann) [inline src/lib/components/embla/Embla.svelte:1]: what do you think about naming the component library agnostic, like `carousel` over `embla`

File: src/lib/components/embla/index.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import EmblaRoot from './Embla.svelte';
+import EmblaSlide from './EmblaSlide.svelte';
+
+export const Embla = Object.assign({
+    Root: EmblaRoot,
+    Slide: EmblaSlide
+});
```
Comment (TorstenDittmann) [inline src/lib/components/embla/index.ts:7]:
```suggestion
export { default as Root } from './Embla.svelte';
export { default as Slide } from './EmblaSlide.svelte';
```

does this work? 👀

File: src/lib/integrations/Carousel.svelte [hunk 4]
```diff
@@ -0,0 +1,69 @@
+<script lang="ts">
+    let carousel: HTMLElement;
+
+    let gap = 32;
```
Comment (TorstenDittmann) [inline src/lib/integrations/Carousel.svelte:4]: this can be a `const` right?

File: src/lib/integrations/index.ts [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+export type ResultType<T> = FuseResult<T>[];
+
+import CarouselRoot from './Carousel.svelte';
+import CarouselSlide from './CarouselSlide.svelte';
+
+export const Carousel = Object.assign({
+    Root: CarouselRoot,
+    Slide: CarouselSlide
+});
```
Comment (TorstenDittmann) [inline src/lib/integrations/index.ts:14]: if the previously mentioned approach works - we can do it here too 👍🏻

File: src/lib/integrations/SvelteFuse.svelte [hunk 6]
```diff
@@ -0,0 +1,17 @@
+<script lang="ts" generics="T">
```
Comment (TorstenDittmann) [inline src/lib/integrations/SvelteFuse.svelte:1]: we might wanna investigate to use server endpoints for searching - might scale better in the long run 👍🏻

File: src/scss/2-resets/_typography.scss [hunk 7]
```diff
@@ -1,4 +1,7 @@
 @use '../abstract' as *;
+html {
+	scroll-behavior: smooth;
+}
```
Comment (TorstenDittmann) [inline src/scss/2-resets/_typography.scss:4]: this will affect all pages, not sure if its intended

**Comment by adityaoberai** [comment]
@TorstenDittmann the page we see under `routes/integrations/article`, can we convert it into a markdoc layout like posts in the blog?

**Comment by adityaoberai** [comment]
> > @TorstenDittmann @thejessewinton  the page we see under `routes/integrations/article`, can we convert it into a markdoc layout like posts in the blog?
> 
> Definitely; on my end it will likely be blocked by Init work, so how soon do you need it?

That's a good point. Gonna take this discussion to Discord.

File: src/lib/components/embla/Embla.svelte (hunk 1 above)
Comment (thejessewinton) [reply src/lib/components/embla/Embla.svelte:168]: This is actually a requirement so `embla` knows which element to attach to.

https://www.embla-carousel.com/get-started/svelte/

File: src/lib/components/embla/Embla.svelte
Comment (thejessewinton) [reply src/lib/components/embla/Embla.svelte:1]: Great, will do!

File: src/lib/components/embla/EmblaSlide.svelte (hunk 2 above)
Comment (thejessewinton) [reply src/lib/components/embla/EmblaSlide.svelte]: 👍🏼

File: src/lib/components/embla/index.ts (hunk 3 above)
Comment (thejessewinton) [reply src/lib/components/embla/index.ts]: It does, just requires multiple imports. Easy change!

File: src/lib/integrations/SvelteFuse.svelte (hunk 6 above)
Comment (thejessewinton) [reply src/lib/integrations/SvelteFuse.svelte]: Was already thinking about this; will throw this in as a busy work task between Init stuff this week.

**Review by ArmanNik — APPROVED** [review]

**Pull request merged**